packages/nomad/loki.hcl

```diff
@@ -0,0 +1,153 @@
+variable "gcp_zone" {
+  type = string
+}
+
+variable "loki_service_port_name" {
+  type = string
+}
+
+variable "loki_service_port_number" {
+  type = number
+}
+
+variable "loki_bucket_name" {
+  type = string
+}
+
+job "loki" {
+  datacenters = [var.gcp_zone]
+  type        = "service"
+
+  priority = 75
```

Comment:
Not entirely related, but based on what you choose the priority?

Reply:
I tried to prioritize the services by:
1. Us knowing what is happening (otel-collector -> logs-collector)
2. Services necessary for sandboxes to be running, in the order in which they are interacted with (API -> proxies, docker-reverse-proxy?)
3. Services necessary for users to know what is happening

Reply:
I think there are some hidden deps (like logs-collector healthchecking loki), that are not accounted for right now.